test/models/data-operation.ts

```diff
@@ -241,9 +241,22 @@ describe('DataOperation', () => {
       url: 'https://example.com/foo',
       temporal: {},
     }];
+    const relatedUrl = {
```

Comment:
I think this test should include a couple of other URLs - one with a Type that isn't 'Color Map' and one that is missing the Type field altogether and then add verifications for both URLs that should be included and URLs that should not be included.

Reply:
Updated, thanks!